src/rgw/setup/rgw_service.py

```diff
@@ -31,14 +31,20 @@ def start(conf: MappedConf, config_file, log_file, motr_trace_dir, index: str =
         try:
             os.environ['M0_TRACE_DIR'] = motr_trace_dir
             cmd = "/usr/bin/radosgw"
-            args = f"-f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
-            args = shlex.split(args)
+            args = f" -f --name client.rgw-{index} -c {config_file} --no-mon-config 2>1"
+            time_stamp = """ | gawk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0 }' """
+            redirect = f" | tee {log_file} "
+            final_cmd = cmd + args + time_stamp + redirect
             sys.stdout.flush()
             sys.stderr.flush()
-            os.execl(cmd, cmd, *args)  # nosec
+            with open('command_file.sh','w') as fd:
+                fd.write(final_cmd)
+            os.execl('/bin/bash', '/bin/bash', 'command_file.sh')
```

Comment:
also codacy says starting process without a shell, given with bash you are starting ?

Reply:
resolved the codac